src/bijectors/corr.jl

```diff
@@ -460,13 +460,8 @@ function _logabsdetjac_inv_corr(Y::AbstractMatrix)
     K = LinearAlgebra.checksquare(Y)
 
     result = float(zero(eltype(Y)))
-    for j in 2:K, i in 1:(j - 1)
-        @inbounds abs_y_i_j = abs(Y[i, j])
-        result +=
-            (K - i + 1) * (
-                IrrationalConstants.logtwo -
-                (abs_y_i_j + LogExpFunctions.log1pexp(-2 * abs_y_i_j))
-            )
+    @inbounds for j in 2:K, i in 1:(j - 1)
+        result += (K - i + 1) * (-LogExpFunctions.logcosh(Y[i, j]))
```

Comment:
Slightly simpler maybe
```suggestion
        result -= (K - i + 1) * LogExpFunctions.logcosh(Y[i, j])
```
or
```suggestion
        result += (i - K - 1) * LogExpFunctions.logcosh(Y[i, j])
```

Reply:
I changed that one just 10 seconds ago 😄